app/components/Header.js

```diff
@@ -167,10 +173,10 @@ const Navigation = ({ onMenuClick = () => {} }) => {
         </ul>
       </section>
       <section className="Header__Nav__Social">
-        <Icon href="https://twitter.com/ReactRally" type="twitter" />
-        <Icon href="https://github.com/react-rally" type="github" />
-        <Icon href="https://instagram.com/reactrally" type="instagram" />
-        <Button href={constants.Links.TICKET_SALES} className="medium">
+        <Icon href="https://twitter.com/ReactRally" type="React Rally Twitter" />
```

Comment:
The `<Icon />` component expects `type` prop to be something that can be used for it's icon resource. I would suggest keeping `type` as it was, and internal to `<Icon />` add `aria-label="React Rally ${type}"`.